app/views/virtual_workshops/_meta_tags.html.haml

```diff
@@ -0,0 +1,17 @@
+- content_for :meta_tags do
+  - title = "Virtual Workshop for #{workshop.chapter.name}"
```

Comment:
Extract title to locale (or reuse existing entry from locale as I think it may be identical)